src/auto/versions.txt

```diff
@@ -1,2 +1,2 @@
-Generated by gir (https://github.com/gtk-rs/gir @ 5836573)
+Generated by gir (https://github.com/gtk-rs/gir @ 83cc9add)
```

Comment:
Last 'd' unneeded
```
diff --git b/src/auto/versions.txt a/src/auto/versions.txt

index 02762ba..90528f6 100644

--- b/src/auto/versions.txt

+++ a/src/auto/versions.txt

@@ -1,2 +1,2 @@

-Generated by gir (https://github.com/gtk-rs/gir @ 83cc9ad)

+Generated by gir (https://github.com/gtk-rs/gir @ 83cc9add)
```

Reply:
Hmm where does that additional d come from? I didn't add it there :)